projects/1.18/assets/croptopia-additions/croptopia_additions/lang/zh_cn.json

```
@@ -0,0 +1,25 @@
+{
+  "item.croptopia_additions.rice_flour": "粘米粉",
+  "item.croptopia_additions.curry_powder": "咖喱粉",
+  "item.croptopia_additions.pad_thai": "泰式炒河粉",
+  "item.croptopia_additions.cherry_soda": "樱桃苏打",
+  "item.croptopia_additions.cotton_candy": "棉花糖",
+  "item.croptopia_additions.cola": "可乐",
+  "advancements.glizzy_gobbler.title": "Glizzy Gobbler",
+  "item.croptopia_additions.rice_noodle": "米粉",
+  "item.croptopia_additions.hot_dog": "热狗",
+  "item.croptopia_additions.kraut_dog": "德式热狗",
+  "item.croptopia_additions.carbonation_machine": "苏打水机",
+  "item.croptopia_additions.frosting": "霜状白糖",
+  "item.croptopia_additions.weiner": "法兰克福香肠",
+  "item.croptopia_additions.sugar_cookie": "甜饼干",
+  "item.croptopia_additions.grape_soda": "葡萄苏打",
+  "advancements.glizzy_gobbler.descr": "吃一个法兰克福香肠",
+  "item.croptopia_additions.bratwurst": "腊肠",
+  "item.croptopia_additions.ramen": "拉面",
+  "item.croptopia_additions.beans_on_toast": "豆酱吐司",
+  "item.croptopia_additions.chicken_curry": "咖喱鸡",
+  "item.croptopia_additions.lemon_lime_soda": "柠檬酸橙苏打",
+  "item.croptopia_additions.orange_soda": "橙汁苏打",
+  "item.croptopia_additions.sauerkraut": "泡菜"
```

```suggestion
  "item.croptopia_additions.lemon_lime_soda": "柠檬苏打",
  "item.croptopia_additions.orange_soda": "橙子苏打",
  "item.croptopia_additions.sauerkraut": "德国酸菜"
```
雪碧七喜原味都叫lemon-lime soda
一般貌似就不把lime带进翻译里的，或者可以叫`柠檬味苏打`

sauerkraut的话，~~sauer就是酸的意思~~好像这样更常见？